common/models/Profiles.php

```diff
@@ -40,7 +40,13 @@ public function rules()
         return [
             [['fullname', 'age'], 'required'],
             [['age'], 'integer'],
-            [['alergias', 'genero'], 'string'],
+            [['alergias'], 'string'],
+            [
+                ['genero'],
+                'match',
+                'pattern' => '(^[mf]$)',
+                'message' =>'Invalid date'
+            ],
```

Comment:
Here it's better to change this to ['alergias', 'in', 'range' => ['M', 'F']], its better for enums and more readable